src/components/Asset/AssetActions/Trade/Swap.tsx

```diff
@@ -167,12 +160,15 @@ export default function Swap({
     asset,
     web3,
     dtItem.token,
+    dtItem.amount,
     baseTokenItem.token,
+    baseTokenItem.amount,
     poolInfo.liquidityProviderSwapFee,
```

Comment:
as we added the full `poolInfo` object as dependency further down, we can clean up this array a bit and remove these 3 `poolInfo.` sub-values.